File: Courses/PHP Web Basics - май 2019/18. Exercise Polymorphism, Interfaces and Abstraction/02.01.Vehicles/Vehicles/Car.php
<?php


namespace Vehicles;


class Car extends VehicleAbstract
{
    const FUELMODIFIER = 0.9;

    public function __construct(float $fuelQuantity, float $fuelConsumption, float $tankCapacity)
    {
        parent::__construct($fuelQuantity, $fuelConsumption, $tankCapacity, self::FUELMODIFIER);
    }
}

File: Courses/C# OOP/Basics/16. Polymorphism Exercise/16. Polymorphism Exercise/01. Vehicles/Vehicles/Car.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace _01._Vehicles.Vehicles
{
    public class Car : Vehicle
    {
        private const double airConditionConsumption = 0.9;

        public Car(double fuelQuantity, double fuelConsumption, double thankCapacity) 
            : base(fuelQuantity, fuelConsumption, thankCapacity)
        {
            FuelConsumption += airConditionConsumption;
        }
    }
}
